web-console/src/components/auto-form/auto-form.tsx

```diff
@@ -84,6 +84,7 @@ export interface Field<M> {
 }
 
 function toNumberOrUndefined(n: unknown): number | undefined {
+  if (n == null) return
```

Comment:
you need a `;` to comply with the code style. 

This section of the web console readme https://github.com/apache/druid/blob/fb08bac01a20723988b0be31452ce14a87a12207/web-console/README.md#developing talks about how to set up your editor to take care of this for you

Reply:
got it, thanks 🙏 will read it through and make the configuration accordingly